nats/main.go

```diff
@@ -44,9 +44,12 @@ See 'nats cheat' for a quick cheatsheet of commands
 	}
 	cli.SetVersion(version)
 
+	defaultConnectionName := "NATS CLI Version " + version
+
 	ncli.Flag("server", "NATS server urls").Short('s').Envar("NATS_URL").PlaceHolder("NATS_URL").StringVar(&opts.Servers)
 	ncli.Flag("user", "Username or Token").Envar("NATS_USER").PlaceHolder("NATS_USER").StringVar(&opts.Username)
 	ncli.Flag("password", "Password").Envar("NATS_PASSWORD").PlaceHolder("NATS_PASSWORD").StringVar(&opts.Password)
+	ncli.Flag("connection-name", "Nickname to use for the underlying NATS Connection (Default is '"+defaultConnectionName+"')").Default(defaultConnectionName).PlaceHolder("NAME").StringVar(&opts.ConnectionName)
```

Comment:
I think listing the default here in the output might be a bit long, I think we can just not print the default here - for most people this will never matter

Reply:
All done.